packages/objection/package.json

```
@@ -0,0 +1,67 @@
+{
+  "name": "@ucast/objection",
+  "version": "0.0.1",
+  "description": "git@github.com:stalniy/ucast.git",
+  "main": "dist/umd/index.js",
+  "module": "dist/es5m/index.js",
+  "es2015": "dist/es6/index.js",
+  "typings": "dist/types/index.d.ts",
+  "scripts": {
+    "prebuild": "rm -rf dist/* && npm run build.types",
+    "build": "rollup -c ../../rollup.config.js -n ucast.objection -g @ucast/core:ucast.core",
+    "lint": "eslint --ext .js,.ts src/ spec/",
+    "test": "mocha -r ts-node/register spec/*",
+    "coverage": "nyc -n src npm run test && nyc report --reporter=lcov"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stalniy/ucast.git"
+  },
+  "keywords": [
+    "objection",
+    "conditions",
+    "query",
+    "builder",
+    "ast"
+  ],
+  "author": "Sergii Stotskyi <sergiy.stotskiy@freaksidea.com>",
```

Don't hesitate to put your name here, people deserve to know their heroes ;)

```suggestion
  "author": "Otávio Augusto Soares <otaviokk@gmail.com>",
  "contributors": [
    { "name": "Sergii Stotskyi", "email": "sergiy.stotskiy@freaksidea.com" }
  ]
```